koku/api/report/serializers.py

```diff
@@ -58,7 +58,37 @@ def validate_field(this, field, serializer_cls, value, **kwargs):
         (ValidationError): if field inputs are invalid
     """
     field_param = this.initial_data.get(field)
+
+    # extract tag_keys from field_params and recreate the tag_keys param
+    tag_keys = None
+    if not kwargs.get('tag_keys') and getattr(serializer_cls,
+                                              '_tagkey_support', False):
+        tag_keys = list(filter(lambda x: 'tag:' in x, field_param))
+        kwargs['tag_keys'] = tag_keys
```

Comment:
This makes the validator create a validation class with the same parameters that `this` was created with. Because `tag_keys` is our custom tagging support, the DRF serializer just shoves it into the `Serializer.data` dict, instead of treating it separately. So, we have to handle that part ourselves.